typeql/language/expressions.feature

```diff
@@ -0,0 +1,504 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+#noinspection CucumberUndefinedStep
+Feature: TypeQL Match Queries with expressions
+
+  Background: Open connection and create a simple extensible schema
+    Given connection has been opened
+    Given connection does not have any database
+    Given connection create database: typedb
+    Given connection open schema session for database: typedb
+    Given session opens transaction of type: write
+
+    Given typeql define
+      """
+      define
+      person sub entity,
+        owns name @key,
+        owns age,
+        owns height,
+        owns weight;
+      name sub attribute, value string;
+      age sub attribute, value long;
+      height sub attribute, value long;
+      weight sub attribute, value long;
+
+      limit-double sub attribute, value double;
+      """
+    Given transaction commits
+    Given connection close all sessions
+
+
+  Scenario: A ValueVariable must have exactly one Assignment constraint
```

Comment:
you should write BDD as language, not code. So this should be a `A value variable must have exactly one assignment constraint` -- please apply throughout.

This follows the "consistency" principle wrt. our code bases